overlay.d/05rhcos/usr/lib/dracut/modules.d/40rhcos-fips/rhcos-fips.sh

```diff
@@ -70,16 +70,22 @@ firstboot() {
       # We need to call zipl with the kernel image and ramdisk as running it without these options would require a zipl.conf and chroot
       # into rootfs
       tmpfile=$(mktemp)
+      optfile=$(mktemp)
       for f in "${tmpsysroot}"/boot/loader/entries/*.conf; do
           for line in title version linux initrd options; do
               echo $(grep $line $f) >> $tmpfile
           done
       done
+
+      echo "Appending 'ignition.firstboot' to ${tmpfile}"
+      sed -e "/^options / s/$/ ignition.firstboot/" -i "$tmpfile"
```

Comment:
Optional: I think we could combine these two steps the way it's done here: https://github.com/coreos/coreos-assembler/blob/0a5bb5180acf4bdcea224a31b2ed1394a3297c6d/src/create_disk.sh#L359.

Though... looking at that `create_disk.sh` code, I think it's also missing a `| cut -d' ' -f2-`. So right now it's passing the `options` keyword into `--parmfile`. I'm guessing `zipl` just treats it as another karg, right? Looks like the original patch where `zipl` support was added does not have this issue: https://github.com/coreos/coreos-assembler/commit/100c2e512ecb89786a53bfb1c81abc003776090d.

Reply:
https://github.com/coreos/coreos-assembler/pull/2233

Reply:
>I'm guessing zipl just treats it as another karg, right?

yep